projects/angular/src/button/button-group/button-group.html

```diff
@@ -10,21 +10,21 @@
 <ng-container *ngIf="menuButtons.length > 0">
   <div class="btn-group-overflow open" [ngClass]="menuPosition" #anchor>
     <button
-      #buttonGroupToggle
+      #menuToggle
       class="btn dropdown-toggle"
       clrPopoverAnchor
       clrPopoverOpenCloseButton
-      (keydown.arrowup)="openButtonGroupMenu($event, InitialFocus.LAST_ITEM)"
-      (keydown.arrowdown)="openButtonGroupMenu($event, InitialFocus.FIRST_ITEM)"
+      (keydown.arrowup)="openMenu($event, InitialItem.LAST)"
+      (keydown.arrowdown)="openMenu($event, InitialItem.FIRST)"
       [attr.aria-controls]="popoverId"
       [attr.aria-expanded]="open"
-      [attr.aria-label]="commonStrings.keys.rowActions"
+      [attr.aria-label]="clrToggleButtonAriaLabel"
```

Comment:
This change seems unrelated to keyboard navigation. I would remove it from here and open a separate PR so that this change is called out as a separate feature in the change log. Otherwise, users won't know we added it unless the dig into the code.